Projects.js

```diff
@@ -0,0 +1,208 @@
+
+    return section.appendChild(project);
+  });
+};
+
+projectMethod();
+
+const modal = document.getElementById('model');
+
+// eslint-disable-next-line no-unused-vars
+function button(id) {
+  modal.style.display = 'flex';
+  const card = projects.find((card) => card.id === id);
+  const project = document.createElement('div');
+  project.classList.add('modelsection');
+  project.classList.add(card.card);
+  project.innerHTML = `<div class="header">
+             <div>
+            <h2>${card.Projectname}</h2>
+             <div class="card_historyy">
+               <div class="card_history_detaill"><p>${card.history.proName}</p></div>
+              <div>
+                     <img src="./images/Counter.svg" alt="dot" />
+              </div>
+                 <div class="card_history_detail">
+                <p>${card.history.tech}</p>
+              </div>
+                <div>
+               <img src="./images/Counter.svg" alt="dot" />
+               </div>
+               <div class="card_history_detail"><p>${card.history.date}</p></div>
+             </div>
+             </div>
+             <a href="#" id="modelcross" class="Menuicon"
+            ><span
+                    id="modelcrossIcon"
+             style="color: black"
+                  class="material-symbols-outlined"
+              >
+                  close
+            </span>
+          </a>
+        </div>
+     <div class="modelimage">
+          <img
+             class="card_imagee"
+              src=${card.image}
+           alt="Tonic Project"
+           />
+           </div>
+           <div class="modeldisc">
+            <div class="card_discription">
+             <p>
+                 ${card.disc}
+               </p>
+          </div>
+            <div class="card_tags">
+             <div class="modeltags">
+          <div class="tag">
+                 <span class="small">${card.tags.html}</span>
+              </div>
+              <div class="tag">
+               <span class="small">${card.tags.css}</span>
+                 </div>
+                <div class="tag">
+              <span class="small"
+                 >${card.tags.javascript}</span
+                       >
+               </div>
+                 </div>
+             <div class="card_action">
+                  <button class="modelbtn" type="button"> See lice <img style="height: 24px; width: 24px; margin-left: 6px;" src="./images/see_live.svg" alt=""></button>
+                   <button class="modelbtn" type="button"> See source <img style="height: 24px; width: 24px; margin-left: 6px;" src="./images/see source.svg" alt=""> </button>
+         </div>
+          </div>
+       </div>`;
+
+  modal.appendChild(project);
+  setTimeout(() => {
+    const span = document.getElementById('modelcrossIcon');
+    // eslint-disable-next-line func-names
+    span.onclick = function () {
+      modal.style.display = 'none';
+      modal.removeChild(project);
+    };
+  }, 50);
+}
```

Comment:
- [ ] According to the project requirements you need to implement a validation that the form is to submit only when the email is all in lower case. You have not implemented this, kindly do. Also don't forget to implement the error message
![image](https://github.com/sheezu900/My_Potfolio/assets/62705947/9abe8fd0-aca8-4536-9905-1869491fd655)

Reply:
sorry this PR was for the popup task.